apps/oxlint/src/lint.rs

```diff
@@ -119,9 +119,24 @@ impl Runner for LintRunner {
 
         let oxlintrc_for_print =
             if misc_options.print_config { Some(oxlintrc.clone()) } else { None };
-        let builder = LinterBuilder::from_oxlintrc(false, oxlintrc)
-            .with_filters(filter)
-            .with_fix(fix_options.fix_kind());
+        let builder = LinterBuilder::from_oxlintrc(false, oxlintrc);
+        // Gracefully report any linter builder errors as CLI errors
+        let builder = match builder {
+            Ok(builder) => builder,
+            Err(err) => match err {
+                LinterBuilderError::UnknownRules { rules } => {
+                    let rules = rules.iter().map(|r| r.full_name()).collect::<Vec<_>>().join("\n");
+                    let error = Error::from(
+                        OxcDiagnostic::warn(format!(
+                        "The following rules do not match the currently supported rules:\n{rules}"
+                    ))
+                        .with_help("Check that the plugin that contains this rule is enabled."),
```

Comment:
This help message is somewhat confusing - it's either:

* we don't support the rule
* the rule is contained in a disabled plugin

Reply:
maybe we should just skip this for now - I foresee in the future us being able to provide more helpful text here like finding which plugin contains this rule. for now though, we can just keep it simple

Reply:
👍